### DISCUSSION TITLE
Did Trivy remove the ability to scan AWS account misconfigurations?

### DISCUSSION THREAD

**Discussion by stevenberry54** [aquasecurity/trivy#7999] [Development]
I was working on automating some Trivy scanning capabilities for my aws account to test misconfigs.  I just started with 0.57.0 and now this is being removed in 0.57.1... what is the logic there?  Is there a new method now?

**Comment by nikpivkin** [comment]
Hi @stevenberry54 !

AWS scanning was put into a separate plugin back in v0.53+ https://github.com/aquasecurity/trivy/discussions/6818

**Reply by stevenberry54** [reply]
I appreciate this response, that makes sense and I was already using the plugin, it's a tuesday monday.  I was in crisis mode 😆 Sorry and thanks for the response!
